src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -29,20 +30,20 @@ public struct Complex : IEquatable<Complex>, IFormattable
         // This value is used inside Asin and Acos.
         private static readonly double s_log2 = Math.Log(2.0);
 
-        private double _real;
-        private double _imaginary;
+        private double m_real;
+        private double m_imaginary;
         
         public Complex(double real, double imaginary)
         {
-            _real = real;
-            _imaginary = imaginary;
+            m_real = real;
+            m_imaginary = imaginary;
```

Comment:
Assuming the upcoming test covers it, I wouldn't think the comment would be necessary. Seems like adding the comments is prone to a situation where one or more comments is omitted, and a reader incorrectly infers that the lack of a comment means the name can change.